requirements.txt

```diff
@@ -2,9 +2,13 @@ pandas
 klayout
 numpy
 matplotlib
-gdsfactory
+gdsfactory==7.7.0
```

Comment:
Is it necessary to fix the gdsfactory version? If yes, how would it be updated in the future?

Reply:
updated reqs.txt to specify v7.7.0